docs/siem/detections/api/lists/lists-api-overview.asciidoc

```diff
@@ -0,0 +1,47 @@
+[[lists-api-overview]]
+== Lists API
+
+Lists can be used with detection rule <<exceptions-api-overview, exceptions>>
+to define values that prevent a rule from generating alerts. Lists are made up
+of:
+
+* *List containers*: A container for values of the same {es}
+{ref}/mapping-types.html[data type] (such as, `IP`, `keyword`, and `text`).
```

Comment:
I would careful about putting the `IP` in caps as the REST API takes it in lower case. The main ones we are showing off in the UI are at the moment are:
<img width="860" alt="Screen Shot 2020-08-04 at 5 23 07 PM" src="https://user-images.githubusercontent.com/1151048/89354857-306b1d80-d677-11ea-9643-bd052572e17a.png">

Which is this exact text that has to be used within the API all lowercase
```ts
ip
ip_range
keyword
text
```

We technically support a few others outside of UI and within the API of:

```ts
boolean
byte
date
date_nanos
date_range
double
double_range
float
float_range
half_float
integer
integer_range
long
long_range
short
```

For the other data types we do not support them enough to mention and then other data types we do not support at all as they don't make any sense from a list use case perspective.